### PR TITLE
Adds config.ini.example

### LINKED ISSUE #40: New serenata_toolbox requires config.ini
Now Rosie needs a `config.ini` in it's root. As Rosie is never uploads anything maybe the `setup` script can just copy [the default from the toolbox](https://github.com/datasciencebr/serenata-toolbox/blob/master/config.ini.example) and rename it…

### PR DESCRIPTION
**Pull request by jtemporal**
This PR closes #40

### CODE REVIEW

**Comment by lipemorais** [comment]
Do you think that would be good to have instructions about it in README?

Something like this code may help to make easier make the config.ini
`cat config.ini.example > config.ini`

**Comment by jtemporal** [comment]
@lipemorais Great idea =)

File: README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ $ ./setup
 $ python rosie.py run
 ```
 
+The toolbox expects to find Amazon credentials in a `config.ini` file. If You get an error of missing `config.ini` you can just copy the `config.ini.example` to a `config.ini` and everything will run smoothly.
```
Comment (cuducos) [inline README.md]: I would add that to that sentence, parethesis inclued:

> … sill run smoothly (don't bother about `AccessKey` and `SercetKey` **unless** you're planning to upload file to S3).

Comment (cuducos) [inline README.md]: BTW many thanks @lipemorais ; )

Comment (jtemporal) [inline README.md]: Added =)

**Pull request merged**